net/gce.sh

```diff
@@ -819,6 +830,10 @@ $(
     cat mount-additional-disk.sh
   fi
 
+  if [[ "$profile" = "true" ]]; then
```

Comment:
I don't think this needs to be optional. Just install it all the time

Reply:
> ok, in this case we can remove `profile` option for `gce.sh`

still need it to pass the compiler flags, right?

Reply:
> > ok, in this case we can remove `profile` option for `gce.sh`
> 
> still need it to pass the compiler flags, right?

We need to pass them only in `net.sh`, for `gce.sh` profile option was primarily to install `perf` and setup permissions to collect some events